pipeline/Makefile

```diff
@@ -350,27 +353,27 @@ ecoli_ref.fastq.bz2.keep.assembly.fa: ecoli_ref.fastq.bz2.keep.abundfilt.keep
 	# 	cp ecoli.37/contigs.fa ecoli_ref.fastq.bz2.keep.assembly.fa
 
 orig_vs_ecoli_assembly.report: ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa
-	dnadiff ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa -p orig_vs_ecoli_assembly
+	dnadiff ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa -prefix orig_vs_ecoli_assembly
 
 ecoli_dn.faa: ecoli_ref.fastq.bz2.keep.assembly.fa
-	/mnt/prokka-1.7/bin/prokka ecoli_ref.fastq.bz2.keep.assembly.fa --outdir ecoli_dn --prefix ecoli_dn --force
+	${HOME}/src/prokka/prokka-* ecoli_ref.fastq.bz2.keep.assembly.fa --outdir ecoli_dn --prefix ecoli_dn --force
```

Comment:
How about using the path `${HOME}/src/prokka/prokka-*/bin/prokka` to match what was there before?